### PR TITLE
#5 meet GitHub

### PR DESCRIPTION
**Pull request by Shvme7**

### CODE REVIEW

File: src/develop.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+console.log("hello world")
+console.log("hello world")
```
Comment (DannyNickson) [inline src/develop.js:2]: nit: Add extra line to the end of file

Comment (DannyNickson) [inline src/develop.js:2]: Remove unnesasary console.log

**Review by DannyNickson — COMMENTED** [review]
Looks good.
Update title and PR description

File: src/develop.js (hunk 1 above)
Comment (Shvme7) [reply src/develop.js]: кк

File: src/develop.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+let a = ("I hope i made it true");
+{
+console.log (a);
+}
```
Comment (DannyNickson) [inline src/develop.js:4]: Please add extra line

File: src/develop.js (hunk 1 above)
Comment (Shvme7) [reply src/develop.js]: ага

Comment (Shvme7) [reply src/develop.js]: ага

File: src/develop.js (hunk 2 above)
Comment (DannyNickson) [inline src/develop.js:4]: nit: We do not need extra "{}" to set a scope. As we use global var a.
```suggestion
console.log (a);
```

File: src/develop.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+let a = ("I hope i made it true");
```
Comment (DannyNickson) [inline src/develop.js:1]: Not shure about "()" here. When we use string type, no need in "()". I belive we can simplify it to
```suggestion
let a = 'I hope i made it true';
```

**Review by DannyNickson — CHANGES_REQUESTED** [review]
I will request changes because occure some codestyle mistakes.
Please follow to the suggestion and code guide lines :smiling_imp:

File: src/develop.js (hunk 3 above)
Comment (Shvme7) [reply src/develop.js:1]: thanks for the tip

File: src/develop.js (hunk 2 above)
Comment (Shvme7) [reply src/develop.js:4]: ok. I'll do better

**Review by DannyNickson — DISMISSED** [review]

File: src/develop.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+let a = 'I hope i made it true';
```
Comment (DannyNickson) [inline src/develop.js:1]: nit:
```suggestion
let a = 'Hope I got it right';
```

File: src/develop.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+let a = 'Hope I got it right';
+console.log (a);
```
Comment (DannyNickson) [inline src/develop.js:2]: As I said, this will not work
Remove unnesesary space
```suggestion
console.log(a);
```

**Review by DannyNickson — CHANGES_REQUESTED** [review]

**Review by DannyNickson — APPROVED** [review]
Do not forget to press `Update Branch` button

**Pull request merged**